.github/workflows/infracost.yml

```diff
@@ -1,170 +0,0 @@
-name: Infracost CI
```

Comment:
Given that we want to rip out infracost completely (https://github.com/nebari-dev/nebari/issues/1817#issuecomment-1557838379), we should probably do it in a separate PR.

Reply:
Rolled back in 59ee8b7d6ac54a086f7bdee2d14f3cead6ec65d6.